packages/@react-aria/menu/docs/useMenuTrigger.mdx

```diff
@@ -121,6 +121,7 @@ function MenuButton(props) {
         <MenuPopup
           {...props}
           domProps={menuProps}
+          aria-label={props.label}
```

Comment:
I think `useMenuTrigger` should already be returning an `aria-labelledby` as part of `menuProps`:

https://github.com/adobe/react-spectrum/blob/8d631f55d1fbe76293c1e6f8a6faf571d61eb0ef/packages/%40react-aria/menu/src/useMenuTrigger.ts#L97

My question is why we are passing `menuProps` to `domProps` here rather than merging with `props`? I think the problem is that the menuProps don't end up being passed to `useMenu` on line 137 because they aren't part of `props`. Then you get the warning because it's missing the `aria-labelledby`. Maybe we can try merging `domProps` into `props` instead?

Reply:
@snowystinger mentioned above that 'we'd want to leave this as domProps so that props get passed to MenuPopup hooks, but domProps do not'

Reply:
that was a my bad, I didn't follow it through useMenuTrigger, sorry for the confusion

Reply:
Thanks! I've made the update to merge domProps into props instead